modules/tinymce/src/plugins/table/test/ts/browser/command/ModifyClassesCommandsTest.ts

```diff
@@ -26,6 +26,11 @@ describe('browser.tinymce.plugins.table.command.ModifyClassesCommandsTest', () =
     events.push(event);
   };
 
+  const setContentAndSelection = (editor: Editor, content: string, selectionStart: number, selectionEnd: number) => {
+    editor.setContent(content);
+    TinySelections.setSelection(editor, [ 0, 0, 0 ], selectionStart, [ 0, 0, 0 ], selectionEnd);
```

Comment:
```suggestion
  const setContentAndSelection = (editor: Editor, content: string, startCellOffset: number, endCellOffset: number) => {
    editor.setContent(content);
    TinySelections.setSelection(editor, [ 0, 0, 0 ], startCellOffset, [ 0, 0, 0 ], endCellOffset);
```
This is more accurate to what's happening here, as atm it's not obvious it must be a cell offset and not something else.